src/main/java/uk/gov/beis/els/api/categories/dishwashers/DishwashersInternetLabelApiForm.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.beis.els.api.categories.dishwashers;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.constraints.NotNull;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.api.common.RescaledInternetLabelApiForm;
+import uk.gov.beis.els.categories.dishwashers.service.DishwashersService;
+
+@Schema(name = "Dishwasher arrow image")
+public class DishwashersInternetLabelApiForm extends RescaledInternetLabelApiForm {
+
+  @Schema(description = "Energy efficiency for dishwasher")
```

Comment:
Nit: I would just set the description to "Energy efficiency class indicator" to keep it consistent with the main HTML label form

Reply:
done